runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -10895,15 +10826,12 @@ J9::Z::TreeEvaluator::VMnewEvaluator(TR::Node * node, TR::CodeGenerator * cg)
        * GPR1 -> Type of Object
        * GPR2 -> Size/Number of objects (if applicable) */
       // We don't need these many registers dependencies as Outlined path will only contain helper call
-      TR::Register *copyEnumReg = enumReg;
-      TR::Register *copyClassReg = classReg;
 
       //////////////////////////////////////////////////////////////////////////////////////////////////////
       ///============================ STAGE 2: Calculate Allocation Size ================================///
       //////////////////////////////////////////////////////////////////////////////////////////////////////
-      // Three possible outputs:
-      // if variable-length array   - dataSizeReg will contain the (calculated) size
-      // if outlined                - tmpReg will contain the value of
+      // Two possible outputs:
+      // if variable-length array   - temp1Reg will contain the (calculated) size
```

Comment:
As suggested previously, rename temp1Reg to something more meaningful and change the comment here as well.